x-pack/plugins/cloud_security_posture/public/pages/findings/layout/findings_layout.tsx

```diff
@@ -147,4 +151,81 @@ export const getFindingsColumns = (): Array<EuiBasicTableColumn<CspFinding>> =>
       </EuiToolTip>
     ),
   },
-];
+] as const;
+
+export const baseFindingsColumns = Object.fromEntries(
+  baseColumns.map((column) => [column.field, column])
+) as Record<typeof baseColumns[number]['field'], typeof baseColumns[number]>;
```

Comment:
the base columns are defined as an array because it's easier to manipulate. 
it's then converted to an object with field name as keys. 

combined with `createColumnWithFilters`, it's used to control which column includes filters.

the gist is that we share column renders but same field can be filterable in one table but not in the other. (example: `resource.id` is filterable in `latest` but not in the other two) 
so the main idea here is to share the basic definition and customize it later in each consumer explicitly 

see `columns` for all tables